app/ui/src/app/core/providers/integration-support-provider.service.ts

```diff
@@ -100,7 +100,10 @@ export class IntegrationSupportProviderService extends IntegrationSupportService
   }
 
   getDeployments(id: string): Observable<IntegrationDeployments> {
-    return this.apiHttpService.setEndpointUrl(integrationEndpoints.deployments, { id }).get();
+    return this.apiHttpService
+      .setEndpointUrl(integrationEndpoints.deployments, { id })
+      .get()
+      .map(response => response['items'] as IntegrationDeployments);
```

Comment:
It's probably possible for 'items' to be undefined if for some reason there's no deployments, may need to do `response['items'] || []` here just to be on the safe side...